crates/cli/src/operator/processor/vault.rs

```diff
@@ -111,12 +111,17 @@ pub async fn process_vault_creation<T: Transport + Clone, P: Provider<T> + Clone
     println!("Deploying the following vaults:");
     println!("{}", serde_json::to_string_pretty(&vault_configs)?);
 
-    let confirm = Confirm::with_theme(&ColorfulTheme::default())
-        .with_prompt("Do you want to proceed with the deployment?")
-        .interact()?;
-    if !confirm {
-        println!("Aborting deployment");
-        return Ok(());
+    println!(
+        "Estimated gas price: {} gwei",
+        util::get_gas_price(&core_instance.deployVaults(vault_configs.clone(), vault_impl)).await?
```

Comment:
Can you assign the call builder and reuse for the actual call as well? also, we should be doing this for all transactions, at least all in the operator part of the CLI